source/slang/slang.cpp

```diff
@@ -3652,6 +3652,14 @@ SLANG_API SlangResult spCompileRequest_getModule(
     return SLANG_OK;
 }
 
+SLANG_API SlangResult spCompileRequest_getSession(
+    SlangCompileRequest* request,
+    slang::ISession** outSession)
+{
+    *outSession = Slang::asInternal(request)->getLinkage();
```

Comment:
This code doesn't seem to be correctly increasing the reference count of the session/linkage.

Reply:
Fixed.